example_cpp_smart_card_client_app/src/chrome_certificate_provider/bridge-backend.js

```diff
@@ -520,6 +512,24 @@ Backend.prototype.rejectedCertificatesCallback_ = function(
       ' certificates: ' + GSC.DebugDump.debugDump(rejectedCertificates));
 };
 
+/**
+ * @param {string} functionName
+ * @param {!Array.<*>} functionArguments
+ * @return {!Array.<*>}
+ */
+function transformFunctionArguments(functionName, functionArguments) {
+  const transformedArguments = functionArguments.slice();
+  if (functionName === 'setCertificates') {
+    // The certificates need to be transformed in order to be recognized by the
+    // API.
+    const certificates = goog.array.map(
+        transformedArguments[0]["clientCertificates"],
+        createClientCertificateInfo);
+    transformedArguments[0]["clientCertificates"] = certificates;
```

Comment:
I think this modifies the key in the original object that was passed into |functionArguments|?